indexer-app/src/cmr/indexer/data/elasticsearch.clj

```diff
@@ -161,6 +161,17 @@
         (info "Existing index set:" (pr-str existing-index-set))
         (info "New index set:" (pr-str expected-index-set))))))
 
+(defn delete-granule-index
+  "Delete an elastic index by name"
+  [context collection-concept-id index]
+  (index-set-svc/remove-deleted-granule-index context idx-set/index-set-id collection-concept-id)
+  (info "Deleting granule index.")
```

Comment:
Flesh this log statement out a bit, needs to be a little more informative to be useful.